prov/rxm/src/rxm_conn.c

```diff
@@ -117,7 +117,11 @@ static void rxm_conn_free(struct util_cmap_handle *handle)
 static struct util_cmap_handle *rxm_conn_alloc(void)
 {
 	struct rxm_conn *rxm_conn = calloc(1, sizeof(*rxm_conn));
-	return rxm_conn ? &rxm_conn->handle : NULL;
+	if (rxm_conn) {
+		dlist_init(&rxm_conn->postponed_tx_list);
+		return &rxm_conn->handle;
+	}
+	return NULL;
```

Comment:
I prefer the style of:
```
if (!rxm_conn)
    return NULL;

/* do all initialization here */
return &rxm_conn->handle;
```
So if more initialization becomes necessary, we don't end up with the entire function indented inside a likely success case.